projects/view/client/rtProject/rtProjectInfoSection.css

```diff
@@ -0,0 +1,215 @@
+.rtProjectInfoSection .section { 
+	max-width: 1024px;
+	clear: both;
+	padding: 0px;
+	margin: 0px;
+	margin-left: auto;
+	margin-right: auto;
+}
+.rtProjectInfoSection .content{ 
+	min-height: 300px;
+	display: table;
+}
+@media only screen and (max-width: 480px) {
+.rtProjectInfoSection	.content{ 
+		min-height: 0px;
+	}	
+}
+.rtProjectInfoSection .content .col{ 
+	display: table-cell;
+	vertical-align: middle;
+	float: none;
+}
+@media only screen and (max-width: 480px) {
+.rtProjectInfoSection	.content .col{ 
+		display: none;
+		min-height: 200px;
```

Comment:
Problem with CSS files is it automatically tabs as tabs and not to spaces; at least in cloud9 ide